CMakeLists.txt

```diff
@@ -46,14 +46,14 @@ endif()
 unset(COVERAGE CACHE)
 
 # LLVM_COVERAGE ==> LIBCELLML_LLVM_COVERAGE
-set(_PARAM_ANNOTATION "Enable OSX coverage testing.")
+set(_PARAM_ANNOTATION "Enable llvm coverage testing.")
```

Comment:
FWIW, it ought to be `LLVM` and not `llvm`.

Reply:
Could have saved some typing if you had just made the change :)

Reply:
Would be too easy. :)